### PR TITLE
feat: geekout template

### PR DESCRIPTION
**Pull request by dhemutton**
This PR contains the addition of a template: GeekOut 2020
Screenshot 2020-10-13 at 12 48 07 AM

### CODE REVIEW

File: src/templates/geekOut2020/geekOut2020.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+export const GeekOut2020: FunctionComponent<TemplateProps<GeekOut2020CertificateTemplate> & {
+  className?: string;
+}> = ({ document, className = "" }) => {
+  return (
+    <>
+      <div css={container} className={className} id="certificate-of-achievement">
+        <div css={textRegion}>
+          <div id="date-range" css={dateRange}>
+            {format(new Date(document.programme.startDate), "d MMM yyyy")} to{" "}
```
Comment (gjj) [inline src/templates/geekOut2020/geekOut2020.tsx:82]: there's an issue with this, see #6

i haven't found the time to fix it - need to find an alternative way of parsing

Comment (dhemutton) [reply src/templates/geekOut2020/geekOut2020.tsx]: Responded to the issue and tested the fix on chrome, firefox, safari!
Screenshot 2020-10-22 at 12 04 36 AM
Screenshot 2020-10-22 at 12 01 27 AM
Screenshot 2020-10-22 at 12 06 07 AM

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"/><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"/><style>html, body {
```
Comment (Nebulis) [inline docs/index.html:1]: whats all those files ?

File: src/templates/geekOut2020/GeekOut2020.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Meta, Preview, Props, Description, Story } from "@storybook/addon-docs/blocks";
```
Comment (Nebulis) [inline src/templates/geekOut2020/GeekOut2020.stories.mdx:1]: better name rc/core/GO-certBase.png ...

File: src/templates/geekOut2020/GeekOut2020.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { object } from "@storybook/addon-knobs";
+import { GeekOut2020 } from "./geekOut2020";
+import { sampleCertificate } from "./fixtures/sample";
+
+<Meta title="MDX|CustomTemplateGeekOut" component={GeekOut2020} />
+
+# CertificateOfAchievement component
```
Comment (Nebulis) [inline src/templates/geekOut2020/GeekOut2020.stories.mdx:8]: name :eyes:

File: src/templates/geekOut2020/__tests__/geekOut2020.test.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  });
+  it("should render with programme name provided by the document", () => {
+    const { getByText } = render(<GeekOut2020 document={sampleCertificate} handleObfuscation={() => void 0} />);
+    expect(getByText("for successfully completing GeekOut 2020!")).toBeInTheDocument();
+  });
+  it("should render with quote", () => {
+    const { getByText } = render(<GeekOut2020 document={sampleCertificate} handleObfuscation={() => void 0} />);
+    expect(getByText("Accept nothing less than your goals!")).toBeInTheDocument();
+  });
+  it("should render with the start and end date provided by the document (format 1: mm-ddd-yyyy)", () => {
```
Comment (Nebulis) [inline src/templates/geekOut2020/__tests__/geekOut2020.test.tsx:24]: why do we have this ? that's not the format expected by the schema

File: src/templates/geekOut2020/__tests__/geekOut2020.test.tsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
+import { GeekOut2020 } from "../geekOut2020";
+import { sampleCertificate } from "../fixtures/sample";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import React from "react";
+
+describe("geekOut2020", () => {
+  it("should render with congratulations", () => {
```
Comment (Nebulis) [inline src/templates/geekOut2020/__tests__/geekOut2020.test.tsx:8]: it's fine to have one test doing multiple assertions when we test one use case only. 

For instance here the first 4 test perform the same test (input is the same) only the assertion change. Then just create one test with 4 assertions => it's the same use case

File: src/templates/geekOut2020/geekOut2020.tsx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+export const GeekOut2020: FunctionComponent<TemplateProps<GeekOutCertificateTemplate> & {
+  className?: string;
+}> = ({ document, className = "" }) => {
+  return (
+    <>
+      <div css={container} className={className} id="certificate-of-achievement">
+        <div css={textRegion}>
+          <div id="date-range" css={dateRange}>
+            {format(new Date(document.programme.startDate.replace(/-/g, "/")), "d MMM yyyy")} to{" "}
```
Comment (Nebulis) [inline src/templates/geekOut2020/geekOut2020.tsx:82]: no need to replace

**Review by Nebulis — APPROVED** [review]

**Pull request merged**